docs/upgrade/v1-5-x-to-v1-6-0.md

```diff
@@ -153,3 +153,40 @@ esac
 The workaround is to replace the above contents generated from gopkg.in/yaml.v3 for `/etc/wicked/scripts/setup_bond.sh` and `/etc/wicked/scripts/setup_bridge.sh` in `/oem/90_custom.yaml` file. Once the file is updated, the upgrade process should resume its progress.
 
 Related issue: [#9033](https://github.com/harvester/harvester/issues/9033)
+
+### 5. Network connectivity issue on secondary vlans for mgmt interface
+
+If users configure secondary vlan interfaces for mgmt interface in addition to the primary vlan configured during installation, then after upgrade from v1.5.x to v1.6.x, the additional vlans will not be part of the `mgmt-br` bridge and `mgmt-bo` due to which network connectivity on those secondary vlans are affected.
+
+For example, consider user having a mgmt interface configured with vlan `2021` during installation and have the following under `/oem/99_vlan-ifcfg.yaml`
+
+```
+stages:
+  initramfs:
+    - name: "Host VLAN interface mgmt-br.353"
+      files:
+        - path: /etc/sysconfig/network/ifcfg-mgmt-br.2113
+          owner: 0
+          group: 0
+          permissions: 384
+          content: |
+            STARTMODE='onboot'
+            BOOTPROTO='static'
+            IPADDR='10.255.113.150/24'
+            VLAN_ID='2113'
+            ETHERDEVICE='mgmt-br'
+            VLAN='yes'
+            DEFROUTE='no'
+
+```
+Users expect to create an additional vlan sub interface on mgmt interface `mgmt-br.2113` and assign IPv4 address and use it for L3 connectivity in addition to the primary `mgmt-br.2021` interface after upgrade to v1.6.x.
+The additional vlan sub interface is created, but `mgmt-br` bridge and `mgmt-bo` will no longer be part of vlan `2113` as after reboot only the primary `vlan-id` will be assigned to the `mgmt-br` bridge and `mgmt-bo` using `/oem/90_custom.yaml`.Starting from v1.6.0, only primary vlan will be added to `mgmt-br` bridge and `mgmt-bo` instead of all 2-4094 vlans.Due to this communication using secondary vlans on mgmt interface is affected.
+Users can check this info using `bridge vlan show` command on the host.
+
+Starting from v1.6.0,Users have to explicitly add the secondary vlans required to the `mgmt-br` bridge and `mgmt-bo` using the following commands (`2113` vlan as an example)
+
+```
+bridge vlan add vid 2113 dev mgmt-br self
+bridge vlan add vid 2113 dev mgmt-bo
+
+```
```

Comment:
Does this serve as a short-term workaround? If the node reboots, the VLAN setting will be lost. Perhaps we can add a warning here and also include a link to the page that discusses how to persist VLAN settings (if such a page exists), since this is the page focused on upgrading known issues, and we don't want to distract people.